### ISSUE TITLE
Some debugger tabs break editor UI at minimal window width

### ISSUE THREAD

**Issue by passivestar** [godotengine/godot#97784]
### Tested versions

v4.4.dev.custom_build [5ccbf6e4c]

### System information

Godot v4.4.dev (5ccbf6e4c) - macOS 14.5.0 - Multi-window, 1 monitor - Metal (Forward+) - integrated Apple M1 Max (Apple7) - Apple M1 Max (10 threads)

### Issue description

Profiler, Visual Profiler and Network Profiler make the window content clip:

https://github.com/user-attachments/assets/02004b50-9ef2-4999-b2b4-d1137e119f78

### Steps to reproduce

Open Profiler/Visual Profiler/Network Profiler and resize the window

### Minimal reproduction project (MRP)

N/A

**Comment by hsvfan-jan** [comment]
On macOS 15.0 with Godot 4.3, I can't reproduce the window width being changed.

I can reproduce a cut-off for the "Inspector" panel when clicking the "Visual Profiler". It makes e.g. the "Forward+ v" button being cut off.

"Profiler" and "Network Profiler" widen the Scene view (the middle panel, not sure what the name is) and shorten the "Inspector" panel. But they don't clip the "Inspector" panel.

**Comment by arkology** [comment]
Should be partially fixed now. See https://github.com/godotengine/godot/pull/102576 and https://github.com/godotengine/godot/pull/102024

**Comment by passivestar** [comment]
Indeed, profiler and visual profiler dont clip in 4.4rc2. This can be closed with #102576
